Makefile

```diff
@@ -28,7 +28,7 @@ checkprefix:
 	  cached_prefix=$(shell $(CMAKE_PRG) -L -N build | 2>/dev/null grep 'CMAKE_INSTALL_PREFIX' | cut -d '=' -f2); \
 	  if ! [ "$(CMAKE_INSTALL_PREFIX)" = "$$cached_prefix" ]; then \
 	    printf "Re-running CMake: CMAKE_INSTALL_PREFIX '$(CMAKE_INSTALL_PREFIX)' does not match cached value '%s'.\n" "$$cached_prefix"; \
-	    $(RM) build/.ran-cmake; \
```

Comment:
`RM` is supposed to be a standard variable defined by make.  What version of make are you using?  What does `make -p` show?